http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/FileUploadResource.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+import io.quarkus.ts.http.restclient.reactive.multipart.MultipartBodyWithMediaType;
+import io.quarkus.ts.http.restclient.reactive.multipart.MultipartBodyWithoutMediaType;
+
+@Path("/upload")
+public class FileUploadResource {
```

Comment:
Reading this shouldn't be `FileUploadResource` the enpoint for the test (and users) and `FileUploadClientResource` for the rest client? (the code inside is fine just naming should be different in my POV, but no need to change it)

Just occure to me when I was going through logic of requests.

Reply:
You're right it could be confusing, so I renamed the class `FileUploadClienttResource` to `FileUploadClientTestResource`